app/controllers/GuidanceController.scala

```diff
@@ -94,6 +103,7 @@ class GuidanceController @Inject() (
             val pageContext = service.getPageContext(evalContext, Some(formData))
             pageContext.page match {
               case page: QuestionPage => Future.successful(BadRequest(questionView(page, pageContext, questionName(path), formWithErrors)))
+              case page: InputPage => Future.successful(BadRequest(inputView(page, pageContext, questionName(path), formWithErrors)))
               case _ => Future.successful(BadRequest(errorHandler.badRequestTemplateWithProcessCode(Some(processCode))))
             }
           },
```

Comment:
Do we need a case for input page around line 120 also? I believe the form error section will handle errors caught by the Play framework. However, there are also errors from failed validation defined in the process.

Reply:
Fixed